Fix bdk-cli default features install (#92)
bitcoindevkit/bitcoindevkit.org: docs/bdk-cli/installation.md

# all features with the async esplora client 
cargo install --git https://github.com/bitcoindevkit/bdk-cli --features=esplora-reqwest,compiler

> # minimal install (electrum and repl features are default)

Reviewer: Sorry needs to be fixed here too:
```suggestion
# minimal install (only repl  feature is on by default)
```

Author: Good catch. Done!
